index.csv

```diff
@@ -118,6 +121,7 @@ jshint,MIT,0.1.8,0.1.9,0.2.0,0.2.1,0.2.2,0.2.3,0.3.0,0.3.1,0.4.0,0.5.0,0.5.1,0.5
 json-loader,MIT,0.1.0,0.1.1,0.1.2,0.1.3,0.1.4,0.1.5,0.1.6,0.1.7,0.1.8,0.2.0,0.5.0,0.5.1
 json-schema,(AFL-2.1 OR BSD-3-Clause),0.2.3
 jsonfile,MIT,0.0.1,1.0.0,1.0.1,1.1.0,1.1.1,1.2.0,2.0.0
+jsonp,MIT,0.0.3,0.0.4,0.2.0,0.2.1
```

Comment:
```suggestion
jsonp,MIT,0.0.3,0.0.4
```

README with License section was added in https://github.com/webmodules/jsonp/commit/dc8722335712d67cc6fb88e70cc092f55f0ba10a ahead of the 0.0.3 release. The author almost certainly just forgot to add a license note to the previous releases, but most anyone's likely to use a newer release, anyway.

Reply:
I'm confused - 0.2.0 comes after 0.0.3, not before it, so it should include both of these, no?

Reply:
I'm confused too, maybe @kemitchell misread `0.2.0` for `0.0.2`? Neither the [v0.0.1 code](https://github.com/webmodules/jsonp/tree/0.0.1) nor the [v0.0.2 code](https://github.com/webmodules/jsonp/tree/0.0.2) on GitHub has the license (matching what's on npm) 


> The author almost certainly just forgot to add a license note to the previous releases, but most anyone's likely to use a newer release, anyway.

Though this makes me think the intended suggestion may have been:

```suggestion
jsonp,MIT,0.0.1,0.0.2,0.0.3,0.0.4,0.2.0,0.2.1
```

which I would be fine with if you prefer. However, given that v0.0.1 and v0.0.2 have a combined weekly download count of (as of writing) 17 I would argue it's not worth including them here.

Reply:
I think I just misread.